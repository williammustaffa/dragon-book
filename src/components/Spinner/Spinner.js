import React from "react";
import { Loader } from "semantic-ui-react";

function Spinner() {
  return (
    <Loader active className="custom-spinner" />
  );
}

export default Spinner