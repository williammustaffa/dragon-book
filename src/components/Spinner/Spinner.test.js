import React from "react";
import Spinner from "./Spinner";
import { render } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"

test("Test Spinner component", () => {
  const { asFragment } = render(<Spinner />);

  // Run tests
  expect(asFragment()).toMatchSnapshot();
});