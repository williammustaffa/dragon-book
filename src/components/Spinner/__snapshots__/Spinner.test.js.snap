// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Spinner component 1`] = `
<DocumentFragment>
  <div
    class="ui active loader custom-spinner"
  />
</DocumentFragment>
`;
